go-controller/hack/update-codegen.sh

```
@@ -21,6 +21,8 @@ olddir="${PWD}"
 builddir="$(mktemp -d)"
 cd "${builddir}"
 GO111MODULE=on go install sigs.k8s.io/controller-tools/cmd/controller-gen@v0.19.0
+ver=$(go -C ${olddir} list -m -f '{{.Version}}' k8s.io/kube-openapi)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)